src/pages/settings/InitialSettingsPage.js

```diff
@@ -74,6 +74,13 @@ const propTypes = {
     /** List of cards */
     cardList: PropTypes.objectOf(cardPropTypes),
 
+    /** Personal Bank Account */
+    personalBankAccount: PropTypes.shape({
+        error: PropTypes.string,
+        shouldShowSuccess: PropTypes.bool,
+        loading: PropTypes.bool,
+    }),
```

Comment:
Probably worth creating a separate personalBankAccountPropTypes file since we're reusing these

Reply:
Done